python/paddle/fluid/dygraph/jit.py

```diff
@@ -55,13 +58,9 @@ def __impl__(*args, **kwargs):
         # Get AST from dygraph function
         dygraph_code = inspect.getsource(dygraph_func)
         root = gast.parse(dygraph_code)
-
         root = DygraphToStaticAst().get_static_ast(root)
-
-        # TODO static_func should a callable from AST, like
-        # static_func = ast_to_func(root)
-        # currently just use dygraph_func
-        static_func = dygraph_func
+        func_name = root.node.body[0].name
```

Comment:
Is it better to a var to store the func_name. Because the content of node.body[0] may be changed in other Transformer.

Reply:
Thanks. Done.